src/lib/contract.ts

```diff
@@ -41,6 +45,83 @@ export async function createContract(
   };
 }
 
+/**
+ * write to warp contract
+ * @params WriteContractProps
+ */
+
+export async function writeContract(params: Types.WriteContractProps) {
+  const warp =
+    params.environment === 'local'
+      ? WarpFactory.forLocal()
+      : params.environment === 'testnet'
+      ? WarpFactory.forTestnet()
+      : WarpFactory.forMainnet();
+
+  let status: number = 400;
+  let statusText: string = 'UNSUCCESSFUL';
+
+  const contract = warp.contract(params.contractTxId).connect(params.wallet);
+
+  const writeContract = await contract.writeInteraction(params.options, {
+    tags: [{ name: 'PermawebJS', value: '1.0.0' }],
```

Comment:
We should update this and any other tags to reflect the current version of the library

Reply:
good catch